Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: "0.2.0"
+appVersion: "0.3.0"
 description: Take mysql backups from any mysql instance to AWS S3. Supports GCP CloudSQL
 name: mysql-backup
 icon: https://avatars0.githubusercontent.com/u/170456
-version: 2.1.4
+version: 2.1.5
```

Comment:
We should bump the minor version (to `2.2.0`) as the change is not (only) a bugfix, but it's adding functionality.

Reply:
you're right. Done